src/mockData.js

```diff
@@ -123,4 +126,6 @@
     }
     ]
   }
-}
\ No newline at end of file
+}
+
+export default mockData;
```

Comment:
^^ Took me forever to realize why I couldn't use mockData in my branch until I saw this lol